<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devices</class>
 <widget class="QWidget" name="devices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>731</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="horizontalLayout_12">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>设备名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string> 设备位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>温度获取</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  湿度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>湿度获取</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>电控箱1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  周市镇</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="temp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_wendu">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>获取温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="humidity">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_shidu">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>获取湿度</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>751</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>    子设备</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>731</width>
        <height>412</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>   设备编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string> 子设备名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string> 子设备位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string> 开关状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string> 设备状态</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>      10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>  开关总阀1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_10">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>      11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>    喷泉1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_11">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>      12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>    喷泉2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_12">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>      13</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>    喷泉3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_13">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>      14</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>    喷泉4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_14">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>      15</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_38">
           <property name="text">
            <string>    喷泉5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_15">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_43">
           <property name="text">
            <string>      16</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_42">
           <property name="text">
            <string>   搅拌机1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_44">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_16">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_47">
           <property name="text">
            <string>      17</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_48">
           <property name="text">
            <string>    水泵1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_49">
           <property name="text">
            <string>   电控箱1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>开启</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_17">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../img/picture.qrc">:/device_close.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="get_state">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>241</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>获取状态</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../img/picture.qrc"/>
  <include location="../img/picture.qrc"/>
  <include location="../img/picture.qrc"/>
 </resources>
 <connections/>
</ui>
